src/ORM/Locator/TableLocator.php

```diff
@@ -243,7 +269,18 @@ protected function _getClassName($alias, array $options = [])
             $options['className'] = Inflector::camelize($alias);
         }
 
-        return App::className($options['className'], 'Model/Table', 'Table');
+        if (class_exists($options['className'])) {
```

Comment:
How about checking if the string contains backslash before doing class_exists() check?